src/UniversalDashboard/Cmdlets/Invoke-UDJavaScript.cs

```diff
@@ -0,0 +1,22 @@
+using NLog;
+using System.Management.Automation;
+using Microsoft.AspNetCore.SignalR;
+
+namespace UniversalDashboard.Cmdlets
+{
+    [Cmdlet(VerbsLifecycle.Invoke, "UDJavaScript")]
+    public class InvokeUDJavaScriptCommand : PSCmdlet
+    {
+		//private readonly Logger Log = LogManager.GetLogger(nameof(SelectUDElementCommand));
+
+        [Parameter(Mandatory = true)]
+		public string jsscript { get; set; }
```

Comment:
Can we rename this to JavaScript?

Reply:
Sure, just on the C# / PowerShell side, or do you want to change it in the JavaScript processing? Commit should be in.

Reply:
On the C#\PS side is fine. Looks good!!